template/Dockerfile.global.sh.twig

```diff
@@ -436,7 +436,13 @@ RUN service mysql start && \
     # make sure assets like logos are ready
     cd /var/www/html && sudo -u www-data sh -c 'php bin/console assets:install' && \
     # add some demo data
+    {% if version_gte(shopware.version, '6.2.0-rc1') %}
```

Comment:
this is AWESOME
thanks
I'm not quite sure if the compare works with -rc1
does it?
otherwise i would be fine with just using 6.2.0 and then I'll merge it :)

Reply:
Hey,
Orca uses `version_compare` under the hood to perform version comparisons.
`version_compare` [is able to compare](https://www.php.net/manual/en/function.version-compare.php) rc versions.